### PR TITLE
Added KWoC Logo

### PR DESCRIPTION
**Pull request by harshkhandeparkar**

### CODE REVIEW

File: src/assets/kwoc_logo.png
Comment (chirag-ghosh) [inline src/assets/kwoc_logo.png:1]: Is this necessary? We can directly use the one in public dir right?

Comment (harshkhandeparkar) [reply src/assets/kwoc_logo.png:1]: I thought files inside `src/` can only import files inside `src/`. I am new to Vite.

Comment (chirag-ghosh) [reply src/assets/kwoc_logo.png:1]: True. But you can use images using absolute url. If you want to import, that's fine. Then you'll need to have them inside src. 

PS - you can make a filder inside src as the public folder. maybe will be useful

**Review by chirag-ghosh — APPROVED** [review]

File: src/assets/kwoc_logo.png
Comment (harshkhandeparkar) [reply src/assets/kwoc_logo.png:1]: Interesting. We'll make changes as we go I think.

**Pull request merged**